code/game/objects/structures/tables_racks.dm

```diff
@@ -182,6 +182,17 @@
 	else
 		return ..()
 
+/obj/structure/table/alt_attack_hand(mob/user)
+	if(user && Adjacent(user) && !user.incapacitated())
+		user.setClickCooldown(CLICK_CD_MELEE)

```

Comment:
Maybe halve the delay? The point is that it's supposed to be spammable and the melee CD is a bit too long.